global.json

```diff
@@ -1,15 +1,15 @@
 {
   "sdk": {
-    "version": "6.0.300-preview.22110.5",
+    "version": "7.0.100-preview.2.22153.17",
```

Comment:
Looks like this is breaking us. @Haplois @nohwnd Shall we update our projects/tests targeting net6 to net7 in our repo or prevent the update here?

Reply:
I fixed it.